config/settings.yml

```diff
@@ -109,6 +109,17 @@
     :additional_regions: {}
   :ems_azure:
     :disabled_regions: []
+    :api_versions:
+      :availability_set: 2017-03-30
```

Comment:
Sounds okay to me, but maybe someone more familiar with Settings like @Fryguy or @carbonin should verify.